libraries/core-react/src/components/Banner/BannerActions.tsx

```diff
@@ -13,11 +11,11 @@ type StyledBannerActionsProps = {
 }
 
 const StyledBannerActions = styled.div<StyledBannerActionsProps>`
-  margin-left: ${enabled.spacings};
+  margin-left: ${tokens.spacings.left};
   grid-column: ${({ placement }) => (placement === 'bottom' ? '1/-1' : 'auto')};
   ${({ placement }) =>
     placement === 'bottom' && {
-      marginTop: enabled.spacings,
+      marginTop: tokens.spacings.left,
```

Comment:
shouldn't this point to a `tokens.spacings.top` property instead of `.left`?

Reply:
Yup!